cdc/owner_test.go

```diff
@@ -124,6 +139,7 @@ func (s *ownerSuite) TestOwnerFlushChangeFeedInfos(c *check.C) {
 }
 
 func (s *ownerSuite) TestOwnerFlushChangeFeedInfosFailed(c *check.C) {
+	log.Warn("Begin", zap.String("Test", "TestOwnerFlushChangeFeedInfosFailed"))
```

Comment:
Don't use the `log` package in the test cases. The `check.C` has an `Log/Logf` interface to output some information. BTW, I don't think we need this log.

Reply:
Okay, I print the log here to troubleshoot why the unit test failed, and then forgot to delete these lines.